backend/src/monarch_py/implementations/solr/solr_query_utils.py

```diff
@@ -52,14 +52,11 @@ def build_association_query(
             query.add_filter_query(" OR ".join([f'object:"{o}" OR object_closure:"{o}"' for o in object]))
     if entity:
         if direct:
-            query.add_filter_query(" OR ".join([f'subject:"{escape(e)}" OR object:"{escape(e)}"' for e in entity]))
+            query.add_filter_query(" OR ".join([f'subject:"{e}" OR object:"{e}"' for e in entity]))
```

Comment:
When I was initially debugging this problem, I noticed that we were escaping and quoting, I thought it made sense change this up to just quote, like we do most other places.